tests/suites/test_suite_cmac.function

```diff
@@ -27,71 +27,56 @@ void mbedtls_cmac_null_args(  )
     mbedtls_cipher_init( &ctx );
 
     /* Test NULL cipher info */
-    TEST_ASSERT( mbedtls_cipher_cmac_update( &ctx, test_data, 16 ) ==
-                                         MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA );
+    TEST_INVALID_PARAM( mbedtls_cipher_cmac_update( &ctx, test_data, 16 ) );
```

Comment:
It appears that the test against the null `cipher_info` validation is only executed for the `..._update()` function and not the others.